### ISSUE TITLE
Gamepad/Controller navigation of UI

### ISSUE THREAD

**Issue by garrynewman** [Facepunch/sbox-issues#954]
- [ ] D-pad moves highlighted controls
- [ ] A clicks
- [ ] Scroll up/down
- [ ] Bring up keyboard UI on text fields

**Comment by DrakeFruit** [comment]
- B for previous menu
- start button should act as escape button
- left joystick and dpad ui navigation
- bumpers to navigate top bar in main menu
- maybe audio feedback for controller navigation?

**Comment by garrynewman** [comment]
Wait this works now?

**Comment by DevulTj** [comment]
Yes, but it's on a stagnant branch and would need to be updated again: https://github.com/Facepunch/sbox/pull/1258
